pipeline/image.py

```diff
@@ -0,0 +1,56 @@
+from collections import defaultdict
+import os
+import datajoint as dj 
+from skimage import io, util
+
+schema = dj.schema(dj.config['custom']['database.prefix'] + 'image')
```

Comment:
Though this works, the recommended way of initializing a schema is like this. Eventually, the lower-case notation will be deprecated. Also, f-strings in general are more performant so I'd always recommend using them when you can.
```suggestion
schema = dj.Schema(f"{dj.config['custom']['database.prefix']}image")
```

Reply:
Changing to `dj.Schema` and using the f-string notation.